choices/choices.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!--Importing Google Icon Font-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!--linking to the reset.css-->
+    <link rel="stylesheet" href="../css/reset.css">
+    <!--Importing materialize.css-->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <!--linking to style.css-->
+    <link rel="stylesheet" href="../css/style.css">
+    <!--Lets browser know website is optimized for mobile-->
+    <meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <!--title as it appears in the the tab-->
+    <title>REÇQUE</title>
+  </head>
+  <body>
+    <!--this represents the left side of the screen and also shows this section's grid layout for different screen sizes-->
+    <div class="row">
+      <div class="col m4 s12">
+        <div class="Mycontainer" >
+          <h3>What type of food would you like to eat? </h3>
+        </div>
+      </div> 
+      
+      <!--this represents the right side of the screen with the clickable icons and also shows this section's grid layout for different screen sizes. In this section we also have the set up onclick events and id's for our functionality-->
+      <div class="col m8 s12">
+        <div class="Iconcontainer2">
+          <div class="vl hide-on-small-only"> </div>
+          <div class="row">
+            <div value="breakfast" class="col l3 m4 s4 notClicked icon"><center><img id="breakfastIcon" class="responsive-img" src="../assets/white icons/breakfast.png" style="width:58px; height:58px;"><p><strong>Breakfast</strong></p></center></div>
```

Comment:
this is primary a style thing...

it is generally not a good practice to have really long lines that force you to scroll to the right 

there are keyboard shortcuts in visual studio code that will automatically do this for you

https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode